requirements-build.txt

```diff
@@ -34,14 +34,15 @@ semantic-version==2.10.0
     # via setuptools-rust
 setuptools-rust==1.8.1
     # via cryptography
-setuptools-scm==8.0.4
+setuptools-scm @ https://files.pythonhosted.org/packages/0e/a3/b9a8b0adfe672bf0df5901707aa929d30a97ee390ba651910186776746d2/setuptools_scm-8.0.4-py3-none-any.whl#sha256=b47844cd2a84b83b3187a5782c71128c28b4c94cad8bfb871da2784a5cb54c4f
```

Comment:
oO
is this.. right? I might need to check if `pybuild-deps` has a bug producing this

Reply:
🤷 I didn't touch this directly. I just let our lock command do its thing. I'll look into it too…

Reply:
this is so weird... setuptools-scm is the exact same version 🤔 
might be related to line 42 pointing that it depends on itself